readthedocs/api/v2/mixins.py

```diff
@@ -35,15 +35,16 @@ def _get_cache_tags(self):
            so any exceptions like 404 should be caught.
         """
         try:
+            tags = []
             project = self._get_project()
             version = self._get_version()
-            tags = [
-                project.slug,
-                get_cache_tag(project.slug, version.slug),
-            ]
-            if self.project_cache_tag:
+            if project:
+                tags.append(project.slug)
+            if project and version:
+                tags.append(get_cache_tag(project.slug, version.slug))
+            if project and self.project_cache_tag:
```

Comment:
we were assuming that project and version were always given, but the static files view isn't attached to a version.

Reply:
This should be a code comment :)